tests/test_data_utils.py

```diff
@@ -87,14 +87,14 @@ def test_get_remote_data_json_404(self, flask_test_client):
                 {"CZZYvE": "Sky builders"},
                 {"CZZYvE": "trust builders"},
             ],
-            "expected_response": "- Sky builders\n     - trust builders",
+            "expected_response": ". Sky builders\n     . trust builders",
         },
         "integer_values": {
             "input_data": [
                 {"GLQlOh": "cost one", "JtwkMy": 4444},
                 {"GLQl6y": "cost two", "JtwkMt": 4455},
             ],
-            "expected_response": "- cost one: 4444\n     - cost two: 4455",
+            "expected_response": ". cost one: 4444\n     . cost two: 4455",
```

Comment:
Don't know if it's part of the requirements, but shouldn't we have a £ sign in here if it's currency?

Reply:
The formatting isn't field specific so adding £ sign could add a prefix to the string but defo worth looking into it